lib/schema/validate.js

```diff
@@ -122,22 +122,41 @@ const processFieldNames = ({ payload = {}, nesting = [], schema = {}, results =
     const value = payload[f];
```

Comment:
Too many overloads of the word `payload` here. And `value` isn't very descriptive.